src/js/components/Game.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Timer from './Game/Timer';
+import Question from './Game/Question';
+import Answers from './Game/Answers';
+
+export default class Game extends React.Component {
+  render() {
+      return (
+           <div id="app-container">
+            <div id="menu-container">
+                <div id="logo">
+                <span id="first-word">Cards</span>
+                <span id="second-word"> for</span>
+                <span id="third-word">Humanity</span>
+                </div>
+                <div id = 'abandon-game-button'>
+                    Abandon Game
```

Comment:
Keep this on one line i.e 
`<div id = 'abandon-game-button'>Abandon Game</div>`